chromium_src/chrome/browser/ui/views/frame/browser_frame_view_layout_linux.h

```diff
@@ -8,9 +8,12 @@
 
 #include "chrome/browser/ui/views/frame/opaque_browser_frame_view_layout.h"
 
-#define GetInputInsets                                  \
-  Unused() { return {}; }                               \
-  int NonClientTopHeight(bool restored) const override; \
+#define GetInputInsets                                                         \
+  Unused() {                                                                   \
```

Comment:
nit: `Unused();`
no need to declare body for "unused" functions.

Reply:
Yup. Done.

Reply:
[2023-02-20T12:53:33.237Z] ld.lld: error: undefined symbol: BrowserFrameViewLinuxNative::MaybeUpdateCachedFrameButtonImages_Unused()

It looks like some platforms the definition is required 🤔

Reply:
ah. looks like virtual functions should be defined, but non-virtual can be left without the body.